android/app/src/main/java/app/organicmaps/search/SearchFragment.java

```diff
@@ -398,9 +398,8 @@ void showSingleResultOnMap(@NonNull SearchResult result, int resultIndex)
     if (RoutingController.get().isWaitingPoiPick())
```

Comment:
What is a `WAITING_POI_PICK` and when is `RoutingController.get().isWaitingPoiPick()` true?

Reply:
When building a route and finish point is defined by search?